plugins/outputs/azure_data_explorer/azure_data_explorer.go

```diff
@@ -154,30 +159,32 @@ func (adx *AzureDataExplorer) writeSingleTable(metrics []telegraf.Metric) error
 func (adx *AzureDataExplorer) pushMetrics(ctx context.Context, format ingest.FileOption, tableName string, metricsArray []byte) error {
 	ingestor, err := adx.getIngestor(ctx, tableName)
 	if err != nil {
+		adx.Log.Error(err)
 		return err
```

Comment:
No! Whenever you return an error back to Telegraf's infrastructure it will be logged. No need for duplicate log messages here.